src/repr/src/strconv.rs

```diff
@@ -180,15 +180,16 @@ where
     //
     // Note that we have to fix up ryu's formatting in a few cases to match
     // PostgreSQL. PostgreSQL spells out "Infinity" in full, never emits a
-    // trailing ".0", and formats positive exponents as e.g. "1e+10" rather than
-    // "1e10". If we need to speed up float formatting, we can look into forking
-    // ryu and making these edits directly, but for now it doesn't seem worth
-    // it.
+    // trailing ".0", formats positive exponents as e.g. "1e+10" rather than
+    // "1e10", and emits a negative sign for negative zero. If we need to speed
+    // up float formatting, we can look into forking ryu and making these edits
+    // directly, but for now it doesn't seem worth it.
 
     match f.classify() {
         FpCategory::Infinite if f.is_sign_negative() => buf.write_str("-Infinity"),
         FpCategory::Infinite => buf.write_str("Infinity"),
```

Comment:
Curiously enough, I get `Infinity` when I use psql, but get `inf` with mzcli.

Reply:
Yeah, `mzcli` has a bunch of weird quirks. It's probably doing the f32 -> text client side rather than server side. Personally I avoid `mzcli`. I find `psql` to be quite a bit more reliable.